cognitive/src/test/scala/com/microsoft/azure/synapse/ml/cognitive/test/CognitiveServicesCommon.scala

```diff
@@ -0,0 +1,7 @@
+package com.microsoft.azure.synapse.ml.cognitive
+
+import com.microsoft.azure.synapse.ml.Secrets
+
+trait CognitiveKey {
+  lazy val cognitiveKey = sys.env.getOrElse("COGNITIVE_API_KEY", Secrets.CognitiveApiKey)
```

Comment:
Centralizing the location of the cognitive key to a common trait so it isn't declared in a specialized file

Reply:
Sounds great bud!